### DISCUSSION TITLE
Unable to add Ingres JDBC driver to setup where no internet connection

### DISCUSSION THREAD

**Discussion by theqaguy1** [dbeaver/dbeaver#21524] [Q&A]
Hi all,
have installed DBeaver on a server with no internet access. I dropped in the JDBC driver as per screenshot but cannot find the class. from reviewing docs, I cannot why it is not picking up.

```
Can't create driver instance
  Error creating driver 'Ingres' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'com.ingres.jdbc.IngresDriver'
  Error creating driver 'Ingres' instance.
Most likely required jar files are missing.
You should configure jars in driver settings.

Reason: can't load driver class 'com.ingres.jdbc.IngresDriver'
    com.ingres.jdbc.IngresDriver
    com.ingres.jdbc.IngresDriver
```

![image](https://github.com/dbeaver/dbeaver/assets/95934168/f09e2bfb-f78f-4b0a-be37-15467fc84608)
getting prompted for drivers but have no internet on server so unable do so.
![image](https://github.com/dbeaver/dbeaver/assets/95934168/1823f3f4-0ef2-4ae1-9fb7-809a7f7c3c53)

Any advice?

DBeaver Version
dbeaver-ce-23.2.2-x86_64-setup community

Operating System
windows 10 pro

Database and driver
JDBC driver: [https://github.com/databendcloud/databend-jdbc version: 0.0.5](https://help.insightsoftware.com/s/article/How-to-install-and-connect-DBeaver-to-Actian-Vector-Database-Accelerator?language=en_US)

**Comment by E1izabeth** [comment]
Could you try removing the red item (licence entry) from the driver libraries list?

**Comment by theqaguy1** [comment]
i added the licence. and then added the Actian JDBC driver. when it asked me to Download it (which I thought i couldn't and don't have internet) i pressed Download the jar was installed. Weird. 

Working fine now.
